components/support/nimbus-fml/fixtures/fe/nimbus_features.yaml

```diff
@@ -27,8 +27,14 @@ features:
         description: This is the map of the player type to a button
         type: Map<PlayerProfile, Button>
         default: {
-          "child": {},
-          "adult": {}
+          "child": {
```

Comment:
So my default validation that I implemented a few weeks ago works 😁 It caught this missing default values for me